libs/autofile/autofile.go

```diff
@@ -126,10 +131,8 @@ func (af *AutoFile) openFile() error {
 		return err
 	}
 	if fileInfo.Mode() != autoFilePerms {
-		// reset file permissions:
-		if err := file.Chmod(autoFilePerms); err != nil {
-			return err
-		}
+		return errors.WithMessage(PermissionsChangedErr,
+			fmt.Sprintf("expected file permissions: %v, got: %v", autoFilePerms, fileInfo.Mode()))
```

Comment:
does it make sense to print the filename too?

Reply:
Yes, now it prints:
```
file: [/var/folders/qf/7nq2274n50v5x9s2y4jcmfgm0000gn/T/permission_test343097720]
		expected file permissions: -rw-------, got: -rwxr-xr-x
```